src/components/date/date.js

```diff
@@ -507,7 +507,12 @@ class Date extends React.Component {
     const datePicker = this.state.open ? this.renderDatePicker() : null;
 
     return (
-      <div className={ this.mainClasses } onClick={ this.handleWidgetClick } { ...tagComponent('date', this.props) }>
+      <div
+        className={ this.mainClasses }
+        onClick={ this.handleWidgetClick }
+        role='presentation'
```

Comment:
It sounds contradictory to add `presentation` to this when it is interactive?

Reply:
My understanding of the `onClick` handler on the div wrapper, is that it's there to catch the interaction with the input it contains. I can have a look at changing this though.

Reply:
I've removed this role